internal/api/rbac/cache.go

```diff
@@ -66,6 +66,10 @@ func (r *cacheImpl) IsEnabled() bool {
 }
 
 func (r *cacheImpl) GetUserProjects(user string, requestAction v1Role.Action, requestScope v1Role.Scope) []string {
+	if permissionListHasPermission(r.guestPermissions, requestAction, requestScope) {
```

Comment:
What does it mean permissionListHasPermission ? is it a strings.Contains like ?

Reply:
Similar yes, however it check for wildcards too